src/pages/home/report/ReportActionPropTypes.js

```diff
@@ -4,17 +4,17 @@ import ReportActionFragmentPropTypes from './ReportActionFragmentPropTypes';
 
 export default {
     // Name of the action e.g. ADDCOMMENT
-    actionName: PropTypes.string.isRequired,
+    actionName: PropTypes.string,
```

Comment:
why the switch to making these prop optional?

Reply:
There was a prop warning in the console that `undefined` was being passed. I looked through the code and there is a spot where the report action passed is `{}` (an empty object) so I just corrected the propTypes here.